app/components/side-bar.hbs

```diff
@@ -0,0 +1,18 @@
+<div class="sidebar">
+  <button class="dropdown-btn" type="button" {{on "click" @toggleDropDown}}>Tasks
+    <i class="fa fa-caret-down"></i>
+  </button>
+  {{#if @showDropDown}}
+    <div class="dropdown-container show">
+      <a href="#" {{on "click" @toggleActiveTasks}}>Active</a>
+      <a href="#" {{on "click" @toggleBlockedTasks}}>Blocked</a>
+      <a href="#" {{on "click" @togglePendingTasks}}>Yet to start</a>
+    </div>
+  {{else}}
+    <div class="dropdown-container close">
+      <a href="#" {{on "click" @toggleActiveTasks}}>Active</a>
+      <a href="#" {{on "click" @toggleBlockedTasks}}>Blocked</a>
+      <a href="#" {{on "click" @togglePendingTasks}}>Yet to start</a>
```

Comment:
Can we apply/remove class based on the condition please, instead of repeating code here

```
 <div class="dropdown-container {{if @showDropDown show hide}}">
```

Reply:
https://guides.emberjs.com/release/components/conditional-content/